README.rdoc

```diff
@@ -69,8 +73,30 @@ The plugin supports the following parameters:
              log message and the detection will start from scratch.
              This is intended as a safeguard against buffering very large
              amounts of data before outputting them to the log stream.
+             Zero means no limit.
              Default: 1000.
 
+[max_bytes]  Maximum number of bytes in a detected exception stack trace.
+             If this maximum number is exceeded, the detected exception stack
+             trace that has been detected so far will be output as a single
+             log message and the detection will start from scratch.
+             This is intended as a safeguard against buffering very large
+             amounts of data before outputting them to the log stream.
+             Zero means no limit.
+             Default: 0.
+
+[stream]  Name of a field in the JSON record that contains the name of a
+          logical log stream within the 'real' log stream.
```

Comment:
Nit: you use single quotes above to delimit field names. Here, `'real'` is just quoting a vague concept, so I'd use double quotes instead. Also below.